.github/workflows/check_mergeability_ghstack.yml

```diff
@@ -2,79 +2,35 @@ name: Check mergeability of ghstack PR
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, edited]
+    types: [opened, synchronize, reopened]
+    branches: [gh/**/base]
```

Comment:
neat!

Though see if we can also check if .github/workflows/check_mergeability_ghstack.yml or .github/scripts/trymerge.py were changed as well.

You should be able to use paths. .github/workflows/check_mergeability_ghstack.yml$|^.github/scripts/trymerge.py$

Reply:
Afaik, paths work in conjunction with branches, e.g. the PR should match BOTH paths AND branches.

A workaround to test the workflow is just to use ghstack (example is this PR).